lib/catalog/create_approval_request.rb

```diff
@@ -2,9 +2,9 @@ module Catalog
   class CreateApprovalRequest
     attr_reader :order
 
-    def initialize(task)
+    def initialize(task, order_item)
```

Comment:
I made a comment but somehow it did not save. My recommendation is not to pass in `order_item`. We do want to ensure task and order_item are coupled.